<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLiteUi</class>
 <widget class="QMainWindow" name="SQLiteUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLiteUi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_open">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Estilo inicial do botão */
QPushButton {
  border: none; /* Sem borda inicialmente */
}

/* Quando passa o mouse por cima */
QPushButton:hover {
  border: 1px solid gray; /* Borda preta */
background-color: rgb(222, 222, 222);	
}


</string>
        </property>
        <property name="text">
         <string>Abrir Banco de Dados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2_new">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/* Estilo inicial do botão */
QPushButton {
  border: none; /* Sem borda inicialmente */
}

/* Quando passa o mouse por cima */
QPushButton:hover {
  border: 1px solid gray; /* Borda preta */
background-color: rgb(222, 222, 222);	
}


</string>
        </property>
        <property name="text">
         <string>Novo Banco de Dados</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_paginas">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Estrutura do Banco</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Navegar">
       <attribute name="title">
        <string>Navegar pelo Banco</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Executar SQL</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
